### DISCUSSION TITLE
Nanase | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1695] [Q&A]
# Nanase | Seeed Studio Wiki

Seeed Studio XIAO SAMD21 with Nanase

https://wiki.seeedstudio.com/Seeeduino-XIAO-by-Nanase/

**Comment by rexvl** [comment]
Hi, I tried DigitalWrite and Single Cycle IOBUS examples but I got only ~6 mhz pulses((

**Reply by eddie3ruff** [reply]
I also tried the IOBUS examples with two different XIAOs and the latest and oldest XIAO board libraries, all with the same results. The quickest pulse width was 500ns, not 1 clock cycle. Could someone test to see what I might be doing wrong?

**Reply by nfs0619** [reply]
Thanks for reaching out! For assistance with your DigitalWrite and Single Cycle IOBUS examples, and to troubleshoot the issue with the pulse width, we recommend contacting our technical support team at techsupport@seeed.io. They can help you with testing and provide guidance for your setup.
